com.hunterwittenborn.Celeste.yml

```diff
@@ -0,0 +1,59 @@
+app-id: com.hunterwittenborn.Celeste
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: celeste
+finish-args:
+  # We need to display on Wayland/X11.
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  # Users may want to sync files from anywhere on their system.
+  - '--filesystem=host'
+  # We need network access in order to sync files to remotes.
+  - '--share=network'
+  # Used for X11 performance as described at https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#f1.
+  - '--share=ipc'
+  # DBus connections we use.
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--talk-name=com.canonical.AppMenu.Registrar'
+  - '--talk-name=com.canonical.indicator.application'
+  # Extra libraries that we've bundled with the Flatpak in order to run.
+  - '--env=LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu:/snap/gnome-3-38-2004/119/usr/lib/x86_64-linux-gnu'
+modules:
+  - name: unsquashfs
+    buildsystem: simple
+    cleanup:
+      - '*'
+    build-commands:
+      - "sed -i 's|#XZ_SUPPORT|XZ_SUPPORT|' squashfs-tools/Makefile"
+      - 'cd squashfs-tools/ && make unsquashfs'
+      - 'install -Dm 755 squashfs-tools/unsquashfs /app/bin/unsquashfs'
+    sources:
+      - type: 'git'
+        url: 'https://github.com/plougher/squashfs-tools'
+        tag: '4.5.1'
+  - name: celeste
+    buildsystem: simple
+    build-commands:
+      # Unpack and install the Snap package and other related files.
+      - 'unsquashfs celeste.snap'
+      - 'cp squashfs-root/* /app/ -R'
+      - 'install -Dm 644 com.hunterwittenborn.Celeste.metainfo.xml /app/usr/share/metainfo/com.hunterwittenborn.Celeste.metainfo.xml'
```

Comment:
instead I would install it in the right location, but later in these step, like after the `rmdir`. This is confusing.